src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class Money {
+
+    public final static int GAME_PRICE = 1000;
+
+    private final int money;
+
+    public Money(int money) {
+        validation(money);
+        this.money = money;
+    }
+
+    public int getPurchaseCount() {
+        return money / GAME_PRICE;
+    }
```

Comment:
객체의 입장에서 생각할때 책임이 좀 더 와닿는다고 해주신 것 처럼 
Money의 입장에서 바라보면 아래와 같은 코드가 좀 더 적합할 수도 있을 것 같아요 :)
아래는 간단한 예시만 들어본 것이니 참고하는데에만 사용해주세요  😄 
```suggestion
    public int divide(int gamePrice) {
        return money / gamePrice;
    }
```